git_hg_sync/repo_synchronizer.py

```diff
@@ -90,6 +96,14 @@ def sync(self, destination_url: str, operations: list[SyncOperation]) -> None:
             except Exception as e:
                 raise RepoSyncError(branch_operation, e) from e
 
+        git_env = {
+            REQUEST_USER_ENV_VAR: request_user,
+        }
```

Comment:
Question: is there any value in propagating all the other environment variables to the Git process by copying the current `os.environ`?:

```
git_env = os.environ.copy()
git_env[REQUEST_USER_ENV_VAR] = request_user
...
```

Reply:
Not really, but OTOH, I don't think that couldn't hurt.